src/toolbar/toolbar-group.jsx

```diff
@@ -19,17 +18,20 @@ function getStyles(props, state) {
 
   const styles = {
     root: {
-      float,
       position: 'relative',
       marginLeft: firstChild ? -marginHorizontal : undefined,
       marginRight: lastChild ? -marginHorizontal : undefined,
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexWrap: 'wrap',
```

Comment:
@nathanmarks : I agree with you completely. I have included it only to make it compatible with the current toolbar design that we have. Even I believe that the user should bear the responsibility of making the thing responsive based on the number of components he uses. 

I will wait for @mbrookes to reply and submit my changes.